Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserThumbnailViewProxyModel.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <QAbstractProxyModel>
+#endif
+
+namespace AzToolsFramework
+{
+    namespace AssetBrowser
+    {
+        class AssetBrowserThumbnailViewProxyModel : public QAbstractProxyModel
+        {
+            Q_OBJECT
+
+        public:
+            explicit AssetBrowserThumbnailViewProxyModel(QObject* parent = nullptr);
+            ~AssetBrowserThumbnailViewProxyModel() override;
+
+            void SetSourceModelCurrentSelection(const QModelIndex& sourceIndex);
+
+            // QModelIndex index(int row, int column, const QModelIndex& parent = QModelIndex()) const override;
```

Comment:
Can remove this and `filterAcceptsRow` below, unless you need them for the additional work still in progress, in which case can ignore this comment.

Reply:
Done